PAMRException: Recipient not connected 4113

Let's suppose ProActive scheduler is used with agents all configured using PAMR. In case of network disconnection, the scheduler will raise an exception that is logged in `Scheduler.log`. This information should be put in correct logging level (e.g debug) and the current warning be less verbose (no stack trace).

Below is an example of the current stack trace:

```
[2015-10-23 17:02:23,829 WARN                  p.remoteobject] Node Source threadpool # 0 #11-thread-12 : unable to contact remote object [pamr://4107/PA-AGENT_NODE] when calling method getActiveObjects
org.objectweb.proactive.core.exceptions.IOException6: Failed to send messsage to pamr://4107/PA-AGENT_NODE
    at org.objectweb.proactive.extensions.pamr.remoteobject.message.PAMRMessage.send(PAMRMessage.java:121)
    at org.objectweb.proactive.extensions.pamr.remoteobject.PAMRRemoteObject.receiveMessage(PAMRRemoteObject.java:93)
    at org.objectweb.proactive.core.remoteobject.RemoteObjectSet.receiveMessage(RemoteObjectSet.java:207)
    at org.objectweb.proactive.core.remoteobject.RemoteObjectAdapter.receiveMessage(RemoteObjectAdapter.java:153)
    at org.objectweb.proactive.core.remoteobject.SynchronousProxy.reify(SynchronousProxy.java:80)
    at pa.stub.org.objectweb.proactive.core.runtime._StubProActiveRuntime.getActiveObjects(_StubProActiveRuntime.java)
    at org.objectweb.proactive.core.runtime.ProActiveRuntimeRemoteObjectAdapter.getActiveObjects(ProActiveRuntimeRemoteObjectAdapter.java:122)
    at org.objectweb.proactive.core.node.NodeImpl.getNumberOfActiveObjects(NodeImpl.java:178)
    at org.ow2.proactive.resourcemanager.nodesource.NodeSource$1.run(NodeSource.java:706)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
    at java.lang.Thread.run(Unknown Source)
Caused by: org.objectweb.proactive.extensions.pamr.exceptions.PAMRException: Recipient not connected 4107
    at org.objectweb.proactive.extensions.pamr.client.AgentImpl$MessageReader.handleError(AgentImpl.java:888)
    at org.objectweb.proactive.extensions.pamr.client.AgentImpl$MessageReader.handleMessage(AgentImpl.java:840)
    at org.objectweb.proactive.extensions.pamr.client.AgentImpl$MessageReader.run(AgentImpl.java:794)
```

--- a/programming-core/src/main/java/org/objectweb/proactive/core/remoteobject/RemoteObjectAdapter.java
+++ b/programming-core/src/main/java/org/objectweb/proactive/core/remoteobject/RemoteObjectAdapter.java
@@ -156,10 +156,11 @@ public class RemoteObjectAdapter implements RemoteObject, Serializable {
             // Log for keeping a trace
             String methodName = message.getMethodName();
             if (!methodName.equals("killRT")) {
-                LOGGER_RO.warn(displayCaller + " : unable to contact remote object " + displayROURI +
-                    " when calling method " + message.getMethodName(), e);
+                LOGGER_RO.warn(displayCaller + ": unable to contact remote object " + displayROURI +
+                    " when calling method " + message.getMethodName());
+                LOGGER_RO.debug(e.getMessage(), e);
             } else if (LOGGER_RO.isDebugEnabled()) {
-                LOGGER_RO.debug(displayCaller + " : unable to contact remote object " + displayROURI +
+                LOGGER_RO.debug(displayCaller + ": unable to contact remote object " + displayROURI +
                     " when calling method " + message.getMethodName(), e);
             }
             return new SynchronousReplyImpl(new MethodCallResult(null, e));

